lib/widgets/simple_chart_widget/time_series.dart

```diff
@@ -0,0 +1,6 @@
+class   TimeSeries {
+  final DateTime time;
+  final int sales;
+
+  TimeSeries(this.time, this.sales);
```

Comment:
I recommend adding @required annotation in classes with has more than 1 parameters, so that in the future, developers would not forget the parameters, causing an error.  It would also warn users before running it that a parameter is missing.

With this annotation instead of simply :
TimeSeries(DateTime(2020, 1, 10), 1)); => 
TimeSeries(time: DateTime(2020, 1, 10), sales: 1));

Reply:
Will do. 💯